### PR TITLE
disabled states and getting started prompt

### PR DESCRIPTION
**Pull request by futurepaul**
wanted to make it more obvious what to do when you first login, and and disable any buttons that are irrelevant when you don't have a mint or a balance

### CODE REVIEW

**Review by benthecarman — COMMENTED** [review]
this add a mint popup is sick, however it pops up everytime i open the app, even if i have mints
image

**Comment by futurepaul** [comment]
> this add a mint popup is sick, however it pops up everytime i open the app, even if i have mints 

dangit thx

File: harbor-ui/src/main.rs [hunk 1]
```diff
@@ -786,6 +797,15 @@ impl HarborWallet {
                         self.active_federation = list.first().cloned();
                     }
 
+                    // Hide the CTA if we have any federations
+                    if !list.is_empty() {
+                        self.show_add_a_mint_cta = false;
+                    }
```
Comment (benthecarman) [inline harbor-ui/src/main.rs:803]: This can just be 

self.show_add_a_mint_cta = list.is_empty()

File: harbor-ui/src/main.rs [hunk 2]
```diff
@@ -786,6 +797,15 @@ impl HarborWallet {
...
+
+                    if !self.has_navigated_to_mints && list.is_empty() {
```
Comment (benthecarman) [inline harbor-ui/src/main.rs:805]: Can do same thing here, don't need the if statement

**Pull request merged**